app/views/_init-count.html

```diff
@@ -0,0 +1,11 @@
+<!--
+  Expects the following variables:
+  podTemplate
+ -->
+
+<span ng-if="pod.spec.initContainers.length" class="text-muted small">
+  <ng-pluralize count="pod.spec.initContainers.length"
+                when="{'1': '&nbsp;Init container','other': '&nbsp;{} Init containers'}">
+  </ng-pluralize>
+  <a ng-href="{{pod | navigateToTabURL:'details'}}">View Details</a>
```

Comment:
For this one, you'll have to take in the entire object, not just the pod template, to create the link. So I'd suggest changing the attribute to `apiObject` / `api-object` and the call the `podTemplate` filter from within the view to get the template.